CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 librdkafka v1.9.3 is a maintenance release:
 
- *  Self-contained static libraries can now be built on Linux arm64 too
+ * Self-contained static libraries can now be built on Linux arm64 too
```

Comment:
get rid of the "too". Just: "Self-contained static libraries can now be built on Linux arm64"

Reply:
and remember the full stop.